app/controllers/UserController.scala

```diff
@@ -0,0 +1,22 @@
+package controllers
+
+import play.api.mvc.{Action, Controller}
+
+class UserController extends Controller {
+
+  def login = Action {
```

Comment:
also remember the good practice of adding `implicit request =>` to the Action bodies

Reply:
this applies to all the methods in here